### ISSUE TITLE
Unexpected behavior for pipe storage cp/mv command

### ISSUE THREAD

**Issue by ekazachkova** [epam/cloud-pipeline#1879]
**Describe the bug**
`pipe storage cp/mv` command performs copy operation but logs error message `Error: 'VersionId'` and returns exit code `1`. This behavior appears for storages with disabled versioning only.

**To Reproduce**
1. Create data storage and disable versioning
2. Try to upload file to storage with CLI: `pipe storage cp /path/to/local/file s3://bucket-name/path/to/file`
3. See result:
```
user:~$ pipe storage cp /path/to/local/file s3://bucket-name/path/to/file
  [##############################]  100% file  209.00/209.00 Kb
Error: 'VersionId'
user:~$ echo $?
1
```

**Expected behavior**
```
user:~$ pipe storage cp /path/to/local/file s3://bucket-name/path/to/file
  [##############################]  100% file  209.00/209.00 Kb
user:~$ echo $?
0
```

**Environment:**
 - Cloud Provider: AWS
 - Version 0.17.0.9062.8b495d36c6810d37ea2e742e1ab9202a690212de

**Comment by tcibinan** [comment]
Should be fixed by bf6f91e634c810e7f36905519149051b52703f35. @ekazachkova Could you please verify?

**Comment by ekazachkova** [comment]
@tcibinan Verified, works well. Thanks!
